sqlx-postgres/src/type_info.rs

```diff
@@ -166,6 +166,11 @@ impl PgTypeInfo {
         self.0.kind()
     }
 
+    /// Returns the OID for this type, if any.
+    pub fn try_oid(&self) -> Option<Oid> {
```

Comment:
```suggestion
    /// Returns the OID for this type, if available.
    ///
    /// The OID may not be available if SQLx only knows the type by name.
    /// It will have to be resolved at runtime which will yield a new and semantically distinct `TypeInfo` instance.
    ///
    /// ### Note
    /// With the exception of [the default `pg_type` catalog][pg_type], type OIDs are *not* stable in PostgreSQL.
    /// If a type is added by an extension, its OID will be assigned when the `CREATE EXTENSION` statement is executed,
    /// and so can change depending on what extensions are installed and in what order, as well as the exact
    /// version of PostgreSQL.
    ///
    /// [pg_type]: https://github.com/postgres/postgres/blob/master/src/include/catalog/pg_type.dat
    pub fn try_oid(&self) -> Option<Oid> {
```